src/assets/scss/main.scss

```diff
@@ -0,0 +1,8 @@
+@import "header";
+@import "variables";
+
+@import "../../../node_modules/bootstrap/scss/bootstrap";
+
+body {
+    background-color: #F7F9F7;
```

Comment:
This should be a variable not a colour code.

Reply:
Alright, I will change this once I've sorted my variables :)